Samples/ExpressJS/index.ts

```diff
@@ -28,10 +28,10 @@ application.post('/prepare', (req, res, next) => {
 
 application.get(
     '/counters',
-    inject(IProjectionStore, 'Logger')(
-        (req, res, next, projections, logger: Logger) => {
+    inject(`IProjectionOf<${DishCounter.name}>`, 'Logger')(
```

Comment:
Not sure if this looks better than just putting in the string. I know this is more type safe, but it adds a bit of strange syntax.

Reply:
We made a nice way to get this type safe